lib/fastlane/plugin/revenuecat_internal/actions/update_hybrids_versions_file_action.rb

```diff
@@ -19,9 +19,25 @@ def self.run(params)
         ios_version = Helper::UpdateHybridsVersionsFileHelper.get_ios_version_for_hybrid_common_version(hybrid_common_version)
         UI.message("Obtained ios version #{ios_version} for PHC version #{hybrid_common_version}")
 
+        billing_client_version = Helper::UpdateHybridsVersionsFileHelper.get_android_billing_client_version(android_version)
+        UI.message("Obtained android billing client version #{billing_client_version} for PHC version #{hybrid_common_version}")
+
         File.open(versions_file_path, 'r+') do |file|
           lines = file.each_line.to_a
-          lines.insert(2, "| #{new_sdk_version} | #{ios_version} | #{android_version} | #{hybrid_common_version} |\n")
+          if lines[0].split('|').length <= 6
+            lines[0] = "#{lines[0].strip} Android Billing Client version |\n"
```

Comment:
Nit: the official name Google uses these days is Play Billing Library. Maybe we should use the same name externally?